XML/TestCases/FunctionalTests-DPDK.xml

```diff
@@ -80,7 +80,7 @@
         <AdditionalHWConfig>
             <Networking>SRIOV</Networking>
         </AdditionalHWConfig>
-        <OverrideVMSize>Standard_D16_v3</OverrideVMSize>
+        <OverrideVMSize>Standard_D16s_v3</OverrideVMSize>
```

Comment:
Why we are moving it from D16 to D16s? Is it requirement by gen2 vms or due to reducing usage?

Reply:
for gen2 image